pandas/tests/io/parser/test_network.py

```diff
@@ -168,6 +168,20 @@ def test_s3_fails(self):
         with pytest.raises(IOError):
             read_csv("s3://cant_get_it/file.csv")
 
+    def test_write_s3_csv_fails(self, tips_df):
+        # GH 32486
+        with pytest.raises(
+            FileNotFoundError, match="The specified bucket does not exist"
+        ):
+            tips_df.to_csv("s3://an_s3_bucket_data_doesnt_exit/not_real.csv")
+
+    def test_write_s3_parquet_fails(self, tips_df):
+        # GH 27679
```

Comment:
Does this need an importorskip, or do we raise prior to importing the engine?

Reply:
Added an importskip. Would raise ImportError if PyArrow or fastparquet isn’t installed.

Reply:
can you add the decorator version instead

Reply:
have done - and changed other importerskip in this file